Source/WebCore/page/skia/MemoryReleaseSkia.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2024 Igalia S.L
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
```

Comment:
Please do not add any LGPL copyright headers.

Reply:
why?

Reply:
We tend to use the LGPL for WPE/GTK port-level sources, and BSD for cross-port ones, IIRC.

Reply:
It was my understanding that all new code should be BSD licensed and it was just the older parts of WebKit that still were LGPL. The [contributing code](https://webkit.org/contributing-code/#develop-your-changes) documentation gives the template for it. Not sure if this is a larger conversation but at the very least no cross port code should be LGPL.

Reply:
For the sake of uniformity (rest of Skia code uses LGPL headers) I propose this PR to continue with LGPL. As for discussion on headers, perhaps we can continue elsewhere? I could create e.g. some draft PR for changing headers and we can discuss there - @donny-dont what do you think?

Reply:
Ok, let me carry that out.

Reply:
> I've checked now and we only use LGPL for `SharedBufferSkia`, and BSD for all other skia files. I'm ok with changing both files to BSD in a follow up.

Sounds good to me as well.